sklearn/model_selection/_split.py

```diff
@@ -1099,7 +1116,8 @@ class _RepeatedSplits(metaclass=ABCMeta):
         Constructor parameters for cv. Must not contain random_state
         and shuffle.
     """
-    def __init__(self, cv, n_repeats=10, random_state=None, **cvargs):
+    @_deprecate_positional_args
+    def __init__(self, cv, *, n_repeats=10, random_state=None, **cvargs):
```

Comment:
```suggestion
    def __init__(self, cv, n_repeats=10, *, random_state=None, **cvargs):
```
but ambivalent.... I think the name says what the parameters are

Reply:
I kinda have a preference for not having it positional since I find the order or `cv` and `n_repeats` arbitrary, if that's okay.